scripts/release/utils/banner.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const { readFile: _readFile } = require('fs');
+const { promisify } = require('util');
+const { resolve } = require('path');
+const readFile = promisify(_readFile);
+
+exports.bannerText = async () => {
+  let BANNER_TEXT = await readFile(resolve(__dirname, 'banner.txt'), 'utf8');
+  if (process.platform === 'darwin') {
```

Comment:
FYI, the latest version of macOS breaks emoji spacing. We're likely removing the fire emoji soon :cry:

Reply:
Sad dayyy... booooo